### PR TITLE
[Yamato] Split build and run

### PR DESCRIPTION
**Pull request by sophiaaar**
### Purpose of this PR
Split up building and running on Yamato so that building can happen on a machine with lots of cores, and the testing can happen on the usual GPU machines.
This is useful to us because it means we won't hog all of Bokken's GPUs while we build players.

---
### Testing status

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/yamato%252Fsplit-build-and-run
Failures are due to bokken

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: .yamato/upm-ci-shadergraph_stereo.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ editors:
     cmdwin: -r %CUSTOM_REVISION%
     cmdposix: -r $CUSTOM_REVISION
 platforms:
-  - name: Windows64
+  - name: Win
```
Comment (MindWrapper) [inline .yamato/upm-ci-shadergraph_stereo.yml]: Why this change is needed?

File: TestProjects/VisualEffectGraph_LWRP/Packages/manifest.json [hunk 2]
```diff
@@ -41,7 +41,9 @@
     "com.unity.modules.video": "1.0.0",
     "com.unity.modules.vr": "1.0.0",
     "com.unity.modules.wind": "1.0.0",
-    "com.unity.modules.xr": "1.0.0"
+    "com.unity.modules.xr": "1.0.0",
+	"com.unity.test-framework.utp-reporter": "0.1.3-preview.17",
```
Comment (MindWrapper) [inline TestProjects/VisualEffectGraph_LWRP/Packages/manifest.json]: wrong indentation

File: .yamato/upm-ci-shadergraph_stereo.yml (hunk 1 above)
Comment (sophiaaar) [reply .yamato/upm-ci-shadergraph_stereo.yml]: Because the jobs reference eachother by name, and some of the links were broken when this name wasn't consistent with the rest of the files

File: TestProjects/VisualEffectGraph_LWRP/Packages/manifest.json (hunk 2 above)
Comment (sophiaaar) [reply TestProjects/VisualEffectGraph_LWRP/Packages/manifest.json]: thanks!
